### PR TITLE
Allow incomplete json in json_extract().

### PR DESCRIPTION
**Pull request by spershin**
Summary: Allow incomplete json in json_extract().

Differential Revision: D58002204

### CODE REVIEW

File: velox/docs/functions/presto/json.rst [hunk 1]
```diff
@@ -133,6 +133,7 @@ JSON Functions
         SELECT json_extract(json, '$.store.book');
 
     Current implementation supports limited subset of JSONPath syntax.
+    JSON string can be incomplete.
```
Comment (mbasmanova) [inline velox/docs/functions/presto/json.rst:136]: Would you include some examples to help illustrate what "incomplete" means?

**Review by mbasmanova — COMMENTED** [review]
CC: @rui-mo @FelixYBW

File: velox/docs/functions/presto/json.rst (hunk 1 above)
Comment (Yuhta) [reply velox/docs/functions/presto/json.rst:136]: Maybe have a more clear definition, something like "JSON string can be just a prefix of a valid JSON".

**Review by amitkdutta — COMMENTED** [review]
Thanks @spershin for looking into incomplete json.

**Pull request closed without merging**